sdk/identity/azure-identity/test/ut/token_credential_impl_test.cpp

```diff
@@ -894,7 +894,6 @@ TEST(TokenCredentialImpl, Diagnosability)
         "Please report an issue with the following details:\n"
         "Token JSON: Access token property ('TokenForAccessing'): string.length=11, "
         "relative expiration property ('TokenExpiresInSeconds'): \"one\", "
-        "absolute expiration property (''): undefined, "
```

Comment:
What's causing this test string to change? I wouldn't have expected this source change requiring a test update like this.

Reply:
These lines on the right: https://github.com/Azure/azure-sdk-for-cpp/pull/5180/files#diff-cc522e423ed15512dfbcccd5a21748c22c948f10b8b779f8201718aed16f98fcR476-R498
It is ok, though. Maybe even better You still can tell from the logs what was going on.